### DISCUSSION TITLE
Provided rules are lost when using validate

### DISCUSSION THREAD

**Discussion by apaofficial** [livewire/livewire#7351] [7. Bugs (archived)]
### Livewire version

3.2.0

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

I have a simple form object without any rules:

```php
class ClaimStatusForm extends Form
{
    public string $state = '';

    public string $note = '';
}
```

When I call validate in my Livewire component with custom rules, the rules get lost and I get an exception:

```php
class ChangeClaimStatus extends Component
{
    public ClaimStatusForm $form;

    public function save()
    {
        $validated = $this->validate([
            'form.state' => 'required',
            'form.note' => 'required',
        ]);
}
```

![image](https://github.com/livewire/livewire/assets/84172517/7d622d3b-657d-48af-a45c-1f8067c1538e)

I suspect that the issue lies here: https://github.com/livewire/livewire/blob/95083c0b61c1910ad28c4628e0dfa9c227eb1c41/src/Features/SupportValidation/HandlesValidation.php#L480

There should be 
```php
$allRules = array_merge($allRules, $form->getRules());
```

Thanks!

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@apaofficial yeah looks like a bug, PRs are welcome! 🙂

**Comment by aeq-dev** [comment]
+1
